05-jupyter/05-jupyter.md

```diff
@@ -24,3 +23,4 @@ Shall you aim for the moon but you were too drank, jupyter has enough (also enou
 
```

Comment:
I think I had some changes to this chapter on Doobies branch

Reply:
Yes, lots to add in this chpater:
1. Kernel
2. IPython
3. Who is Jovyan
4. Websockets